portal_objects/metaworkflows/SNV_annot.yaml

```diff
@@ -0,0 +1,275 @@
+#       define all arguments for the pipeline here
+#############################################################
+input:
+
+  input_vcf:
+    argument_type: file.vcf_gz
+    dimensionality: 1
+  
+  reference:
+    argument_type: file.fa
+    files:
+      - complete-reference-fasta@hg38
+  
+  regions:
+    argument_type: file.txt
+    files:
+      - 10Mb-genomic-regions@hg38
+  
+  vep:
+    argument_type: file.vep_tar
+    files:
+      - vep@101b
+  
+  clinvar:
+    argument_type: file.vcf_gz
+    files:
+      - clinvar@20201101
+
+  dbnsfp:
+    argument_type: file.dbnsfp_gz
+    files:
+      - dbnsfp@4.1a
+
+  maxent:
+    argument_type: file.tar_gz
+    files:
+      - maxent@20040421
+
+  spliceai_snv:
+    argument_type: file.vcf_gz
+    files:
+      - spliceai-snv@1.3
+
+  spliceai_indel:
+    argument_type: file.vcf_gz
+    files:
+      - spliceai-indel@1.3
+
+  gnomad:
+    argument_type: file.vcf_gz
+    files:
+      - gnomad-genome@3.1
+
+  gnomad2:
+    argument_type: file.vcf_gz
+    files:
+      - gnomad-exome@2.1.1
+
+  CADD_snv:
+    argument_type: file.tsv_gz
+    files:
+      - cadd-snv@1.6
+
+  CADD_indel:
+    argument_type: file.tsv_gz
+    files:
+      - cadd-indel@1.6
+
+  phylop30bw:
+    argument_type: file.BigWig
+    files:
+      - phylop30way-mammalian@hg38
+  
+  phylop100bw:
+    argument_type: file.BigWig
+    files:
+      - phylop100way-vertebrate@hg38
+
+  phastc100bw:
+    argument_type: file.BigWig
+    files:
+      - phastcons100way-vertebrate@hg38
+
+  gene_panel:
+    argument_type: file.tsv_gz
+    files:
+      - gene_panel_hartwig@hg38
+  
+  somatic_hotspot:
+    argument_type: file.vcf_gz
+    files:
+      - somatic_hotspot_mutations@hg38
+
+  version: 
+    argument_type: parameter.string
+    value: "101"
+
+  assembly:
+    argument_type: parameter.string
+    value: "GRCh38"
+
+
+## Workflows and dependencies ###############################
+#     Information for the workflows and their dependencies
+#############################################################
+workflows:
+
+  ## Workflow definition #####################
+  #  splitter
+  ############################################
+  splitter:
+
+    ## Workflow arguments ##############
+    ####################################
+    input:
+
+      input_vcf:
+        argument_type: file.vcf_gz
+        scatter: 1
+
+    ## Output ##########################
+    ####################################
+    output:
+
+      indel_snv_pass:
+        description: VCF containing SNVs and INDELs of the input VCF
+        file_type: raw VCF
+
+      sv_bnd_pass:
+        description: VCF containing SVs of the input VCF
+        file_type: raw VCF
+
+    ## EC2 Configuration to use ########
+    ####################################
+    config:
+        instance_type: 
+         - t3.large
+         - t3.xlarge
+         - c5.4xlarge
+        ebs_size: 1.6x
+        EBS_optimized: True
+        spot_instance: False
```

Comment:
this probably should the True once it's tested

Reply:
will switch it off everywhere after the final testing